src/@utils/assetConvertor.ts

```diff
@@ -14,7 +14,8 @@ export async function transformAssetToAssetSelection(
   const algorithmList: AssetSelectionAsset[] = []
 
   for (const asset of extendedAssets) {
-    const algoComputeService = getServiceByName(asset, 'compute')
+    const algoComputeService =
```

Comment:
based on what it is checking, the variable is not exclusive to the compute service anymore, so should be renamed to e.g.:

```suggestion
    const algoService =
```

Reply:
makes sense now 😃